packages/fxa-settings/src/pages/Pair/Unsupported/index.tsx

```diff
@@ -0,0 +1,43 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React from 'react';
+import { RouteComponentProps } from '@reach/router';
+import { FtlMsg } from 'fxa-react/lib/utils';
+import CardHeader from '../../../components/CardHeader';
+import Banner, { BannerType } from '../../../components/Banner';
+import { usePageViewEvent } from '../../../lib/metrics';
+import { HeartsBrokenImage } from '../../../components/images';
+import { REACT_ENTRYPOINT } from '../../../constants';
+type PairUnsupportedProps = { error?: string };
+export const viewName = 'pair-unsupported';
+
+const PairUnsupported = ({
+  error,
+}: PairUnsupportedProps & RouteComponentProps) => {
+  usePageViewEvent(viewName, REACT_ENTRYPOINT);
+
+  return (
+    <>
+      {error && (
```

Comment:
What would you think about adding a comment about localizing the error message, as you did in another PR? (just so we remember later on)